src/api/uploads/FolderUploadNode.js

```diff
@@ -117,18 +117,20 @@ class FolderUploadNode {
             const data = await this.createFolder();
             this.folderId = data.id;
         } catch (error) {
-            errorEncountered = true;
-            errorCode = error.code;
             // @TODO: Handle 429
             if (error.code === ERROR_CODE_ITEM_NAME_IN_USE) {
                 this.folderId = error.context_info.conflicts[0].id;
             } else if (isRoot) {
+                errorEncountered = true;
```

Comment:
It looks like this variable wouldn't be used if `isRoot` is true due to line 139. Is this needed? Should we test this case, if so?

Reply:
i can remove the duplicates. by removing 120-121, this reveals the two variables were being set in cases where they weren't being used